invenio_jsonschemas/views.py

```diff
@@ -46,9 +48,17 @@ def create_blueprint(state):
     def get_schema(schema_path):
         """Retrieve a schema."""
         try:
-            return send_from_directory(state.get_schema_dir(schema_path),
-                                       schema_path)
+            schema_dir = state.get_schema_dir(schema_path)
         except JSONSchemaNotFound:
             abort(404)
+        if request.args.get(
+            'refs',
+            current_app.config.get('JSONSCHEMAS_REPLACE_REFS'),
+            type=int
+        ):
+            schema = {'$ref': request.base_url}
```

Comment:
I guess I can always create a `Loader` that does something like [this](https://github.com/inspirehep/inspire-schemas/blob/master/inspire_schemas/utils.py#L35) but I liked the idea of just using `invenio` components so that this `replace_refs` would work in the same way that calling `Record.validate` does.